src/zh/komiic/src/eu/kanade/tachiyomi/extension/zh/komiic/Payload.kt

```diff
@@ -0,0 +1,49 @@
+package eu.kanade.tachiyomi.extension.zh.komiic
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Payload<T>(
```

Comment:
use simple classes, no need for data classes in most cases

Reply:
That cause these vars unused warning. is that ok?
![image](https://github.com/user-attachments/assets/42d60b1f-2802-448f-8029-fdd9be5a10fc)

Reply:
Can make them private if they aren't being accessed from outside, and unused warnings on the payload is fine